From 7471e16ce146692e3ceff809c2f7a20e11cf2c54 Mon Sep 17 00:00:00 2001
From: Thorsten Blum <thorsten.blum@linux.dev>
Date: Sun, 21 Dec 2025 13:47:13 +0100
Subject: [PATCH 13319/15333] orangefs: Remove commented out code in
 find_cached_xattr

The code has been commented out since 2017 - remove it.

Signed-off-by: Thorsten Blum <thorsten.blum@linux.dev>
Signed-off-by: Mike Marshall <hubcap@omnibond.com>
---
 fs/orangefs/xattr.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/fs/orangefs/xattr.c b/fs/orangefs/xattr.c
index b6d116302de4..885fd3bd5a3d 100644
--- a/fs/orangefs/xattr.c
+++ b/fs/orangefs/xattr.c
@@ -72,11 +72,6 @@ static struct orangefs_cached_xattr *find_cached_xattr(struct inode *inode,
 	if (hlist_empty(h))
 		return NULL;
 	hlist_for_each_entry_safe(cx, tmp, h, node) {
-/*		if (!time_before(jiffies, cx->timeout)) {
-			hlist_del(&cx->node);
-			kfree(cx);
-			continue;
-		}*/
 		if (!strcmp(cx->key, key))
 			return cx;
 	}
-- 
2.55.0

